frontend/src/views/WheresMyGene/components/GetStarted/style.ts

```diff
@@ -1,29 +1,56 @@
 import styled from "@emotion/styled";
-import { EXPANDED_WIDTH_PX } from "src/components/common/SideBar";
+import { FILTERS_PANEL_EXPANDED_WIDTH_PX } from "src/components/common/SideBar";
 import { HEADER_HEIGHT_PX } from "src/components/Header/style";
+import {
+  CONTENT_WRAPPER_LEFT_RIGHT_PADDING_PX,
+  CONTENT_WRAPPER_TOP_BOTTOM_PADDING_PX,
+} from "src/components/Layout/style";
+import { LEGEND_MARGIN_BOTTOM_PX } from "../../style";
 import { CELL_INFO_SIDEBAR_WIDTH_PX } from "../CellInfoSideBar/style";
+import {
+  X_AXIS_CHART_HEIGHT_PX,
+  Y_AXIS_CHART_WIDTH_PX,
+} from "../HeatMap/utils";
+import { LEGEND_HEIGHT_PX } from "../InfoPanel/components/Legend/style";
 
 export const Header = styled.h1`
   margin-bottom: 12px;
   font-size: 36px;
   font-weight: bold;
 `;
 
+// Matches padding/gap between heat map chart and x/y axis components
+const GAP_PX = 5;
+
+const OFFSET_FROM_TOP_PX =
```

Comment:
Calculating offset from top for GetStarted component instead of using an arbitrary number